### ISSUE TITLE
Revisit all current dashboards to add new and remove unneeded to make them more useful for catching bugs.

### ISSUE THREAD

**Issue by krasi-georgiev** [prometheus/test-infra#186]
Brian tries to provide some useful ones in 
https://grafana.com/orgs/robustperception/dashboards

Need to monitor current bugs, talk to the maintainers and make the dashboards more useful to catch bugs during the bench tests. 

Current dashboards are not bad, but there is room of improvement.
http://prombench.prometheus.io/grafana

**Comment by krasi-georgiev** [comment]
Few ideas:

 - [ ] add rules
 - [ ] add metrics for the time and size retention to make sure it works as expected

**Comment by brian-brazil** [comment]
Or wait for me to update my dashboards, which already have those ;)

**Comment by krasi-georgiev** [comment]
lets put this on hold for now and will revisit in 1-2 months.

**Comment by geekodour** [comment]
Adding the new dashboards should become easier after #293 is merged.
